modules/preview/rest/preview-endpoint.php

```diff
@@ -46,7 +47,7 @@ public static function register_routes() {
 				'expiration'      => [
 					'required'          => true,
 					'validate_callback' => function ( $param ) {
-						$expiration_options = VIP_Workflow::instance()->preview->get_link_expiration_options();
+						$expiration_options = Preview::get_link_expiration_options();
```

Comment:
This is what inter-module communication should look like moving forward, with proper intellisense and no dynamic properties.